config.py

```diff
@@ -26,7 +26,7 @@ class Config(object):
     DEBUG = False
     ADMINS = frozenset([__YOUR_EMAIL__])
     TESTING = False
-    HOST = '127.0.0.1'
+    HOST = '0.0.0.0'
```

Comment:
The `HOST` in the `Production` class is already set to `0.0.0.0`. If you can't run `Production` mode, try adding a `Docker` class which will in turn give you a `Docker` mode.